src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureFlagTracing.cs

```diff
@@ -63,11 +72,19 @@ public void UpdateFeatureFilterTracing(string filterName)
             }
         }
 
+        public void UpdateMaxVariants(int currentFlagTotalVariants)
```

Comment:
```suggestion
        public void NotifyMaxVariants(int currentFlagTotalVariants)
```

It may not update anything.